master/usage/decommissioning-a-node.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Decommissioning a Node
+---
+
+If you are decommissioning a host that runs calico/node (e.g. because it
+is being taken out of service, or is being renamed) it is necessary to
+explicitly delete the associated [Node resource][Node resource reference]
+to remove any data associated with that node.  In environments where nodes
+will be added and removed regularly this clean up should be included in the
+cluster management system. This page provides instruction on how to remove
+a host's associated node data.
+
+> **Notes**
+>
+> - Removing a Node resource will also remove the Workload Endpoint
+>   resources and other sub configuration items associated with that Node.
+> - Prior to removing a Node resource from the datastore the calico/node
+>   container should be stopped and it should be ensured it will not be
+>   restarted.  If calico/node is restarted the configuration information
```

Comment:
"the configuration" is misleading - it sounds like starting calico/node will cause all of your deleted data to be added back in - which isn't true.

Maybe omit the second sentence.  We already say "don't restart it"